src/contract/method.rs

```diff
@@ -102,10 +102,20 @@ impl<T: Transport, R: Detokenize> MethodBuilder<T, R> {
         self.view().call()
     }
 
-    /// Sign (if required) and send the transaction.
+    /// Sign (if required) and send the method call transaction.
     pub fn send(self) -> SendFuture<T> {
         self.tx.send()
     }
+
+    /// Send a transaction for the method call and wait for confirmation.
+    /// Returns the transaction receipt for inspection.
+    pub fn send_and_confirm(
```

Comment:
Cool, although this seems like its own feature ;-)

Reply:
It was needed for the example.... :zipper_mouth_face: